eahub/config/settings.py

```diff
@@ -193,11 +193,16 @@ class DjangoEnv(Enum):
 ACCOUNT_USERNAME_REQUIRED = False
 
 # search
+IS_ENABLE_ALGOLIA = env.str("IS_ENABLE_ALGOLIA", default=True)
```

Comment:
I think it needs to be added to build_settings.py at well 😅

Reply:
done